tlslite/constants.py

```diff
@@ -733,10 +841,16 @@ class CipherSuite:
     sha256Suites.append(TLS_RSA_WITH_NULL_SHA256)
     sha256Suites.append(TLS_DH_ANON_WITH_AES_128_CBC_SHA256)
     sha256Suites.append(TLS_DH_ANON_WITH_AES_256_CBC_SHA256)
+    sha256Suites.append(TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256)  # unsupported
+    sha256Suites.append(TLS_ECDH_ECDSA_WITH_AES_128_CBC_SHA256)  # unsupported
+    sha256Suites.append(TLS_ECDH_RSA_WITH_AES_128_CBC_SHA256)  # unsupported
```

Comment:
sha256Suites include only ciphers that use SHA-256 both as an HMAC and PRF. GCM ciphersuites use SHA-256 only as an PRF.